crazyflie_driver/src/crazyflie_server.cpp

```diff
@@ -112,6 +123,52 @@ class CrazyflieROS
     m_thread.join();
   }
 
+  /**
+   * Service callback which transmits a packet to the crazyflie
+   * @param  req The service request, which contains a crtpPacket to transmit.
+   * @param  res The service response, which is not used.
+   * @return     returns the result of the sendPacket call to the crazyflie
+   */
+  bool sendPacket (
+    crazyflie_driver::sendPacket::Request &req,
+    crazyflie_driver::sendPacket::Response &res)
+  {
+    uint8_t length = req.packet.size;
+    uint8_t data[length+1];
```

Comment:
Does this compile? In my understanding C++ (even 11) requires the size of arrays to be known at compile time?